src/rules/arrayBracketSpacingRule.ts

```diff
@@ -19,7 +19,7 @@ export class Rule extends Lint.Rules.AbstractRule {
       - \`"always"\`requires one or more spaces or newlines inside array brackets
 
       The second option is an object for exceptions to the \`"never"\` option:
-      
+
```

Comment:
Very good point. The only reason I noticed that one was because of my IDE. I just checked `gulp-tslint` and it seems that there is a PR to get it working (https://github.com/panuhorsmalahti/gulp-tslint/pull/113/files) with tslint 5.0. I'll update once they release it.